interface/src/index.html

```diff
@@ -70,12 +70,22 @@
                 Download
               </a>
             </li>
+            <li ui-sref-active="active" ng-if="!user">
+              <a href="/tribe_client">
+                Login with Tribe
```

Comment:
Is it okay if we just call this "Login" and then mention that people can use their Tribe ID if they already have one? It could be confusing otherwise if people think they must be Tribe users to use ADAGE.